pkg/kwokctl/runtime/kind/kind.go

```diff
@@ -127,4 +127,5 @@ type BuildKindConfig struct {
 	SchedulerExtraVolumes         []internalversion.Volume
 	ControllerManagerExtraArgs    []internalversion.ExtraArgs
 	ControllerManagerExtraVolumes []internalversion.Volume
+	ExtraLogVolumes               []internalversion.Volume
```

Comment:
Can We use `ExtraVolumes` for this instead?

https://github.com/kubernetes-sigs/kwok/blob/72824baf0362ba6c309554379185c6b180b1fb07/pkg/kwokctl/runtime/kind/kind.go#L47